src/OV/System.cpp

```diff
@@ -113,6 +113,13 @@ OpenvarioSetRotation(DisplayOrientation orientation)
   WriteConfigFile(map, Path("/boot/config.uEnv"));
 }
 
+void
+OpenvarioSetLanguage(const char* lang)
+{
+  Run("localectl", "set-locale", lang);
+  Run("export", fmt::format("LANG={}", lang).c_str());
```

Comment:
This has zero effect, even if this function call were able to invoke a shell built-in (which it does not). So you spawn a shell, use `export` to set an environment variable in this process, and then the process exits. The environment change is gone.